contrib/hamilton/contrib/user/zilto/nixtla_statsforecast/__init__.py

```diff
@@ -0,0 +1,203 @@
+import logging
+from typing import Callable, Optional
+
+logger = logging.getLogger(__name__)
+
+from hamilton import contrib
+
+with contrib.catch_import_errors(__name__, __file__, logger):
+    import matplotlib
+    import pandas as pd
+    from statsforecast import StatsForecast
+    from statsforecast.models import (
+        _TS,
+        AutoARIMA,
+        CrostonClassic,
+        DynamicOptimizedTheta,
+        HistoricAverage,
+        HoltWinters,
+        SeasonalNaive,
+    )
+    from utilsforecast.evaluation import evaluate
+    from utilsforecast.losses import mse
+
+
+def raw_dataset() -> pd.DataFrame:
```

Comment:
Hmm -- I'm not sure this should be in this module? Maybe it should just take in the dataset as an input, and then one woudln't have to pass it in as overrides? It would be more generic?